docs/constants/navigation.js

```diff
@@ -145,6 +146,7 @@ const home = [
     makePage('deploy/app-stores-metadata.mdx'),
     makePage('deploy/send-over-the-air-updates.mdx'),
   ]),
+  makeSection('Monitor', [makePage('monitoring/monitor-your-app.mdx')]),
```

Comment:
```suggestion
  makeSection('Monitor', [makePage('monitoring/monitor-your-app.mdx')]),
```

This URL structure is bit repetitive (thinking in terms of SEO and a good structure). How about we update it to `monitoring/tools` or `monitoring/services`, considering all are service providers listed on this page.